### PR TITLE
Add Polkadot Bridgehub

### PR DESCRIPTION
**Pull request by lazam**
New Polkadot Common Good Parachain: Bridgehub

### CODE REVIEW

File: packages/apps-config/src/endpoints/productionRelayPolkadot.ts [hunk 1]
```diff
@@ -624,6 +624,17 @@ export const prodParasPolkadotCommon: EndpointOption[] = [
...
+  },
+  {
+    info: 'polkadotBridgeHub',
+    paraId: 1002,
+    providers: {
+      Parity: 'wss://polkadot-bridge-hub-rpc.polkadot.io'
+    },
+    text: 'BridgeHub',
+    ui: {
+      logo: nodesBridgeHubBlackSVG
```
Comment (serban300) [inline packages/apps-config/src/endpoints/productionRelayPolkadot.ts:636]: We should use the `nodesBridgeHub` logo here. The `nodesBridgeHubBlack` one is for Kusama.

**Comment by serban300** [comment]
> Do we merge as-is or change the logo?
> 
> Sadly I dropped the image initially uploaded when I cleaned up since it was unused - sorry about that. Having said that, it should be simple to add - https://github.com/polkadot-js/apps/tree/master/packages/apps-config#endpoints
> 
> TL;DR
> 
>     1. Add file to apps-config/ui/logos/nodes
> 
>     2. Run `yarn build:images`
> 
>     3. Change import as suggested above

However you prefer. I'm also ok with merging as-is, and then changing the logo in another PR. 

Attaching the image to this message just in case it's needed.

BridgeHub

**Review by jacogr — APPROVED** [review]
Merging as-is. Follow-up PRs with an updated logo welcome.
